src/components/taskDetails/TaskDates.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Details from './Details';
+import styles from './task-details.module.scss';
+import { TASK_EXTENSION_REQUEST_URL } from '@/constants/url';
+import convertTimeStamp from '@/helperFunctions/convertTimeStamp';
+
+interface TaskDatesProps {
+    isEditing: boolean;
```

Comment:
wouldn't it be better to name isEditingMode?

Reply:
if I changed it to `isEditingMode` I have to changed it into many files not just here

Reply:
I think isEditing also make sense here @tejaskh3